include/bluetooth/services/bas.h

```diff
@@ -12,8 +13,7 @@
 extern "C" {
 #endif
 
-void bas_init(void);
-void bas_notify(void);
+int bas_notify(void);
```

Comment:
Is it an API function? Then why there is no comment at all?

Reply:
Because there wasn't any to begin with and the PR is focused on something else.

Reply:
There should then at least be a follow-up patch (either in this PR, or another one after it) that prefixes these appropriately. Everything under include/bluetooth must at least have a bt_* prefix. I'll let you add more namespacing if you feel like it's appropriate (e.g. bt_gatt_*)

Reply:
@jhedberg I have prefixed it with `bt_gatt_` and added a small doxy section.